package.json

```diff
@@ -163,27 +163,27 @@
                 {
                     "command": "extension.ccUndoCheckout",
                     "group": "cc@3",
-                    "when": "vscode-clearcase:enabled"
+                    "when": "editorTextFocus && !inOutput && vscode-clearcase:enabled"
```

Comment:
I still can find the `inOutput` flag. Not in the doc or in the Default Keybindings file.